cmd/catchpointdump/net.go

```diff
@@ -76,6 +78,51 @@ var netCmd = &cobra.Command{
 			return fmt.Errorf("network or round not set")
 		}
 
+		if rawDump {
```

Comment:
this new code is very similar to the code below. Why not to make a new function `loadAndRawDump` and call it conditionally instead of `loadAndDump` below? The address selection and loop over addresses is the same, the difference only in tar opening and `rawDumpCatchpointStream` calling.